lib/evm_database_ops.rb

```diff
@@ -5,8 +5,8 @@
 
 class EvmDatabaseOps
   include Vmdb::Logging
-  BACKUP_TMP_FILE = "/tmp/miq_backup".freeze
-  DUMP_TMP_FILE   = "/tmp/miq_pg_dump".freeze
+  BACKUP_TMP_FILE     = "/tmp/miq_backup".freeze
+  DUMP_TMP_FILE       = "/tmp/miq_pg_dump".freeze
```

Comment:
Why was this changed?

Reply:
I had another line in there that required aligning these lines.  When I removed it I failed to undo the indentation. I'll fix.